### DISCUSSION TITLE
Use lexical outside react

### DISCUSSION THREAD

**Discussion by ceberterreche** [facebook/lexical#5467] [Q&A] [answered]
Hello,

I would like to know if it is possible to use Lexical in a nodejs environment outside of React.

In my case, I store the editorState in a database and I want to update this content in the backend. For example, change a specific range to bold or get the plain text from the stringify editorState that I have stored in my database.

If it's possible can you give me a code example ? 

thx

**Comment by pauldmps** [comment, marked as answer]
Lexical is implemented as a plain JS library so it can be used in any JS environment.

For Node.js look at the doumentation for headless mode:
https://lexical.dev/docs/packages/lexical-headless
